<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>w_scr_dish_data_collection_dialog</class>
 <widget class="QDialog" name="w_scr_dish_data_collection_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,1">
     <item>
      <widget class="lds_tableView" name="tableView"/>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>4</number>
       </property>
       <item row="19" column="1">
        <widget class="lds_toolbar_button" name="pushButton_serach">
         <property name="text">
          <string>查询</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton</string>
         </property>
        </widget>
       </item>
       <item row="14" column="1" rowspan="2">
        <widget class="QLineEdit" name="lineEdit_username">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="14" column="0" rowspan="2">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>用户账号</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="11" column="1" rowspan="2">
        <widget class="QLineEdit" name="lineEdit_cashier">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_mallid"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>服务器</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>结束</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1" rowspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit_txdate_yyyymmdd">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_txtime_hhmmss">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>商场编号</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="1" rowspan="2">
        <widget class="QLineEdit" name="lineEdit_ch_uploadmemo">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>交易货号</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_storecode"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>网页</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="11" column="0" rowspan="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>收银员</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="0" rowspan="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>时间</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1" rowspan="2">
        <widget class="QLineEdit" name="lineEdit_ch_uploadflag">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="21" column="1" rowspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>1000</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_itemcode"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>交易铺号</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="lineEdit_service_ip"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_service_port"/>
         </item>
        </layout>
       </item>
       <item row="7" column="0" rowspan="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>状态备注</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0" rowspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>状态</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="16" column="0" colspan="2">
        <widget class="QLabel" name="label_13">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>2</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>line_div_h</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_service_submit_path"/>
       </item>
       <item row="17" column="1">
        <widget class="lds_datetimeEdit" name="dateTimeEdit">
         <property name="displayFormat">
          <string>yyyy-MM-dd hh:mm:ss</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <widget class="lds_datetimeEdit" name="dateTimeEdit_2">
         <property name="displayFormat">
          <string>yyyy-MM-dd hh:mm:ss</string>
         </property>
        </widget>
       </item>
       <item row="20" column="1">
        <widget class="lds_toolbar_button" name="pushButton_repair">
         <property name="text">
          <string>repair</string>
         </property>
         <property name="outer_stylesheet" stdset="0">
          <string>pushbutton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton_save">
       <property name="text">
        <string>保存</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="lds_toolbar_button" name="pushButton">
       <property name="text">
        <string>退出</string>
       </property>
       <property name="outer_stylesheet" stdset="0">
        <string>pushbutton</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lds_toolbar_button</class>
   <extends>QPushButton</extends>
   <header>lds_toolbar_button.h</header>
  </customwidget>
  <customwidget>
   <class>lds_datetimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>lds_datetimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>lds_tableView</class>
   <extends>QTableView</extends>
   <header>lds_tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
